src/devices/GoPiGo3/Models/GrooveInputOutput.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.GoPiGo3.Models
+{
+    /// <summary>
+    /// The setup of the Groove element
+    /// </summary>
+    public enum GrooveInputOutput
```

Comment:
Grove, not Groove

Reply:
Yes, I will change in all the project. I took the (bad) habit to call all the Grove sensors Goove. Changing everywhere for next commit